types/jquery/jquery-tests.ts

```diff
@@ -35,12 +35,36 @@ function JQueryStatic() {
 
         // $ExpectType JQuery<HTMLElement>
         $('<p></p>');
+        
+        // $ExpectType JQuery<HTMLParagraphElement>
+        $<HTMLParagraphElement>('<p></p>');
+        
+        // $ExpectType JQuery<HTMLParagraphElement>
+        $<HTMLParagraphElement>('<p></p>', document);
+        
+        // $ExpectType JQuery<HTMLParagraphElement>
+        $<HTMLParagraphElement>('p');
+        
+        // $ExpectType JQuery<HTMLParagraphElement>
+        $<HTMLParagraphElement>('p', document);        
 
         // $ExpectType JQuery<HTMLElement>
         $(new HTMLElement());
 
         // $ExpectType JQuery<HTMLElement>
         $([new HTMLElement()]);
+        
+        // $ExpectType JQuery<HTMLElement>
```

Comment:
Should be `// $ExpectType JQuery<HTMLParagraphElement>`.

Reply:
Thanks, overlooked this type.